airflow/providers/google/cloud/hooks/cloud_memorystore.py

```diff
@@ -572,7 +571,7 @@ def _append_label(instance: cloud_memcache.Instance, key: str, val: str) -> clou
     @staticmethod
     def proto_message_to_dict(message: proto.Message) -> dict:
         """Helper method to parse protobuf message to dictionary."""
-        return json.loads(message.__class__.to_json(message))
+        return message.__class__.to_dict(message)
```

Comment:
Do we need this method at all? I think we can replace these methods with direct calls.

Reply:
I was thinking about it as well. Please check last updates to the PR.